approved/j750/flow_control_flow.txt

```diff
@@ -17,17 +17,21 @@ DFF 1.1	Flow Table
 						Test	pgm1	pgm1						Fail									flag-true	t4_C2A232F_PASSED				
 						Test	pgm1	pgm1						Fail									flag-true	t4_C2A232F_PASSED				
 						logprint	Test_that_if_ran_works																					
-						Test	pgm	pgm			10			Fail	t5_C2A232F_RAN													
+						Test	pgm	pgm			10			Fail														
+						flag-true	t5_C2A232F_RAN																					

```

Comment:
Logically the same as the original, this reflects the new way of handling if_ran conditions by not changing the failing criteria for the target test and then setting a flag to indicate we got to this part of the flow immediately afterwards.